index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css?family=Poppins:100,100italic,200,200italic,300,300italic,regular,italic,500,500italic,600,600italic,700,700italic,800,800italic,900,900italic" rel="stylesheet" />
+    <script src="https://kit.fontawesome.com/2df075da03.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+  <header class="nav">
+    <nav>
+      <p>My Logo</p>
+      <img src="./Button.png" alt="menu button">
+    </nav>
+  </header>
+  <section class="headline-content">
+    <h1>I'm Thiago <br>Glad to see you!</h1>
+    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <h3>LET'S CONNECT</h3> 
+    <div class="socials">
+        <a href="#"><i class="fab fa-twitter fa-2x"></i></a>
+        <a href="#"><i class="fab fa-linkedin-in fa-2x"></i></a>
+        <a href="#"><i class="fab fa-medium-m fa-2x"></i></a>
+        <a href="#"><i class="fab fa-github fa-2x"></i></a>
+        <a href="#"><i class="fab fa-angellist fa-2x"></i></a>
+    </div>
```

Comment:
When we are dealing with multiple links like in this instance, for better practices and semantics, we should group them in a ul tag and each list item(icons) inside the ul element should be as a li 😁